app/src/test/java/io/hstream/testing/ClusterKillNodeTest.java

```diff
@@ -46,18 +68,40 @@ public void teardown() throws Exception {
   }
 
   @Test
-  void listStreamAfterKillNode() {
+  void listStreamAfterKillNodes() {
+    String stream = randStream(hStreamClient);
     hServers.get(0).close();
-    hStreamClient.listStreams();
+    hServers.get(1).close();
+    Assertions.assertEquals(stream, hStreamClient.listStreams().get(0).getStreamName());
```

Comment:
~~Are you sure that `listStreams` guarantees the order (in time) of streams?~~ The test will use a brand-new test environment so it is OK.